crates/orchestrator/src/cli/cron/event_bridge.rs

```diff
@@ -0,0 +1,22 @@
+use clap::Args;
+
+/// Parameters used to config AWS SNS.
+#[derive(Debug, Clone, Args)]
+#[group()]
+pub struct AWSEventBridgeCliArgs {
+    /// Use the AWS Event Bridge client
+    #[arg(long)]
+    pub aws_event_bridge: bool,
+
+    /// The name of the S3 bucket.
+    #[arg(env = "MADARA_ORCHESTRATOR_EVENT_BRIDGE_TARGET_QUEUE_NAME", long, default_value = Some("madara-orchestrator-event-bridge-target-queue-name"))]
+    pub target_queue_name: Option<String>,
+
+    /// The cron time for the event bridge trigger rule.
+    #[arg(env = "MADARA_ORCHESTRATOR_EVENT_BRIDGE_CRON_TIME", long, default_value = Some("10"))]
```

Comment:
when we do `--help`, clap is supposed to show a help message for all args. where do we entire that help text? like here I was thinking to display that this time inseconds

Reply:
Added help text